src/Language/PureScript/Pretty/JS.hs

```diff
@@ -182,14 +161,13 @@ conditional = mkPattern match
 accessor :: (Emit gen) => Pattern PrinterState JS (gen, JS)
 accessor = mkPattern match
   where
-  match (JSAccessor _ prop val) = Just (emit prop, val)
+  match (JSAccessor _ prop val) = Just (emit $ T.pack $ codePoints prop, val)
```

Comment:
I'm actually not too familiar with this part of the compiler so I'm not sure. Perhaps this field should remain as `Text`, not `PSString`, if there's no need to accommodate strings which aren't valid UTF-16 (which presumably there isn't with static property access like this)?